sensor/extrinsic_reflector_based_calibrator/scripts/metrics_plotter_node.py

```diff
@@ -0,0 +1,265 @@
+#!/usr/bin/env python3
+
+# Copyright 2023 Tier IV, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import math
+
+import matplotlib.pyplot as plt
+import rclpy
+from rclpy.node import Node
+from std_msgs.msg import Float32MultiArray
+
+
+class Plotter:
+    def __init__(self):
+        self.fig, self.axes = plt.subplots(nrows=2, ncols=2, figsize=(8, 6))
+        self.subplot0 = self.axes[0, 0]
+        self.subplot1 = self.axes[0, 1]
+        self.subplot2 = self.axes[1, 0]
+        self.subplot3 = self.axes[1, 1]
+        plt.gcf().canvas.set_window_title("Metrics plotter")
+
+        self.color_bo = "bo-"
+        self.color_go = "go-"
+        self.color_b = "b"
+        self.color_g = "g"
+
+        (
+            self.num_of_reflectors,
+            self.cv_distance_error,
```

Comment:
The name `cv` is quite misleading. Please avoid abbreviations when possible

Reply:
All of the cv changed to crossval by this commit https://github.com/tier4/CalibrationTools/pull/145/commits/82d940da6f5dfdebeea22fd0649d4526e63a8c7f.